runtime/src/test/java/com/dylibso/chicory/testing/TestModule.java

```diff
@@ -17,6 +18,14 @@ public static TestModule of(File file) {
         return new TestModule(file);
     }
 
+    public static TestModule of(File file, ModuleType moduleType) {
+        if (moduleType == ModuleType.TEXT) {
```

Comment:
This change is going to affect only the execution of the tests in this codebase, we should probably revisit this piece of code: https://github.com/dylibso/chicory/blob/5dadeaa21241f3f0d1ada659fb9670490f1a7e30/runtime/src/main/java/com/dylibso/chicory/runtime/Module.java#L493